backend/fleet_management/tests/test_cars.py

```diff
@@ -0,0 +1,42 @@
+from urllib.parse import urlencode
+
+from django.contrib.auth.models import Group
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from fleet_management.constants import Groups
+from fleet_management.factories import CarFactory, UserFactory
+from fleet_management.serializers import CarSerializer
+
+
+class CarsApiTest(APITestCase):
+    def setUp(self):
+        self.url = reverse("cars")
+        self.user = UserFactory()
+        self.user.groups.set(Group.objects.filter(name=Groups.Driver.name))
+        self.cars = [
```

Comment:
Why not
```
self.cars = CarFactory.create_batch(3, country=self.user.country)
```
and then in `test_search_by_plate`
```
url_params = urlencode({"search": self.cars[0].plate})
```
?

Reply:
Totaly right, fixed.